azure-devops/azext_devops/dev/pipelines/commands.py

```diff
@@ -77,3 +85,8 @@ def load_build_commands(self, _):
         # basic release commands
         g.command('list', 'release_definition_list', table_transformer=transform_release_definitions_table_output)
         g.command('show', 'release_definition_show', table_transformer=transform_release_definition_table_output)
+
+    with self.command_group('pipelines runs artifact', command_type=runOps) as g:
```

Comment:
I am a bit confused on this..
This command can download build artifacts as well?
If yes then should there be `pipelines build runs artifact` command group as well?
c: @geverghe  for inputs

Reply:
I am waiting mainly for this to close

Reply:
@gauravsaralMs the way I've implemented it, it doesn't matter whether it is a pipeline or a build (they are the same APIs anyway right?). Is it worth implementing ```pipelines build runs artifact``` though? Seems redundant since they would be identical.

Reply:
agree @mitchdenny . internally both can be same.
but the question is more around "is it build artifact and pipeline artifact same from a user perspective ?".
cc: @geverghe

Reply:
My apologies for the delayed response. As @gauravsaralMs pointed out, `pipelines runs artefact` and `build artifact` are two different concepts segregated from a user experience/concept. Although behind the scenes we are using the same build API, it is a temporary fix till the pipelines APIs are out. Therefore in summary: 
1. `pipelines runs artifact` and `build artifact` (no runs here since build and build-definition are separate sub groups) are different conceptually. 
2. It is worth implementing `pipelines build runs artifact` and switch it out to the pipelines API once it is done.

Reply:
We've had a side conversation with the Pipelines team about this. The net result is that we are going to just have the one ```runs``` command tree under ```pipelines```. The unified pipeline experience is arriving in the very short term and there is no point adding additional ```runs``` command trees when they are going to be relatively short lived - especially since this is the first CLI download experience we have for Pipeline-based artifacts.